lib/transformers/summary.js

```diff
@@ -42,11 +42,13 @@ module.exports = ({schemaKey, schema, options, data, fullUiSchema} = {}) => {
             }
         });
 
-        summaryLists += `<h2 class="govuk-heading-l">${summaryList.title}</h2>
+        if (rows.length > 0) {
+            summaryLists += `<h2 class="govuk-heading-l">${summaryList.title}</h2>
               {{ govukSummaryList({
                 classes: 'govuk-!-margin-bottom-9',
                 rows: ${JSON.stringify(rows, null, 4)}
```

Comment:
just a thought: Change this to:
`rows: JSON.stringify(rows)`
Is this uneeded formatting on the json?
i did a quick perf test and it is 20% slower than just doing my suggestion above.

Reply:
I'm not sure if it applies here but in some situations the whitespace is required: https://github.com/CriminalInjuriesCompensationAuthority/q-transformer/blob/b0732d18c9f6bc3770c0c6e3ec33304d704b7587/lib/helpers/nunjucks/index.js#L55

Reply:
Yeah, the spacing is required for nunjucks to render this correctly.